e2e-tests/cypress/e2e/pages/insurance/check-your-answers/index.js

```diff
@@ -0,0 +1,5 @@
+import eligibilityCheckYourAnswers from './eligibility';
+
+export {
+  eligibilityCheckYourAnswers,
```

Comment:
How about `checkYourAnswersEligibility`? Will be easier to read when we have other answer pages